### PR TITLE
docs: add JSON type for the gRPC SDKs

### LINKED ISSUE #1276: update the doc that gRPC protocol already support JSON type

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

close #1276 
refer to: https://github.com/GreptimeTeam/greptimedb/issues/4872

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/user-guide/ingest-data/for-iot/grpc-sdks/go.md [hunk 1]
```diff
@@ -217,6 +217,61 @@ affected, err := cli.CloseStream(ctx)
 
 </div>
 
+<div id="ingester-json-type">
+
+In the [low-level API](#low-level-api),
+you can specify the column type as `types.JSON` using the `AddFieldColumn` method to add a JSON column.
+Then, insert JSON data as a string value.
+
+```go
```
Comment (nicecui) [inline docs/user-guide/ingest-data/for-iot/grpc-sdks/go.md:226]: @daviderli614 PTAL, if there are errors, please provide the correct code, thx

File: docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md [hunk 2]
```diff
@@ -304,6 +304,54 @@ LOG.info("Write result: {}", result);
 
 </div>
 
+<div id="ingester-json-type">
+
+In the [low-level API](#low-level-api),
+you can specify the column type as `DataType.JSON` using the `addField` method to add a JSON column.
+Then, insert JSON data as a string value.
+
+```java
```
Comment (nicecui) [inline docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md:313]: @fengjiachun PTAL, if there are errors, please provide the correct code, thx

File: docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md [hunk 3]
```diff
@@ -304,6 +304,54 @@ LOG.info("Write result: {}", result);
...
+TableSchema sensorReadings = TableSchema.newBuilder("sensor_readings")
+        // The code for creating other columns is omitted
+        // ...
+        // specify the column type as JSON        
+        .addField("attributes", DataType.JSON)
+        .build();
+
+// ...
+// Add JSON data as a string value
+sensorReadings.addRow(<other-column-values>... , "{\"location\":\"factory-1\"}");
```
Comment (nicecui) [inline docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md:324]: @fengjiachun I consider if we can provide common code for converting a map to a string, the user experience here will be more friendly.

**Review by fengjiachun — APPROVED** [review]
LGTM

File: docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md (hunk 3 above)
Comment (fengjiachun) [reply docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md:324]: https://github.com/GreptimeTeam/greptimedb-ingester-java/issues/54

**Review by daviderli614 — APPROVED** [review]

**Pull request merged**